website/docs/guides/duckdb-qs.md

```diff
@@ -0,0 +1,309 @@
+
+    ```shell
+
+    python3 -m venv venv
+    source venv/bin/activate
+    python3 -m pip install --upgrade pip
+    python3 -m pip install -r requirements.txt
+    source venv/bin/activate
+
+    ```
+    </Expandable>
+
+    <Expandable alt_header="Example for Windows" >
+
+    ```shell
+
+    python -m venv venv
+    venv\Scripts\activate.bat
+    python -m pip install --upgrade pip
+    python -m pip install -r requirements.txt
+    venv\Scripts\activate.bat
+
+    ```
+
+    </Expandable>
+
+    <Expandable alt_header="Example for Windows PowerShell" >
+
+    ```shell
+
+    python -m venv venv
+    venv\Scripts\Activate.ps1
+    python -m pip install --upgrade pip
+    python -m pip install -r requirements.txt
+    venv\Scripts\Activate.ps1
+
+    ```
+    </Expandable>
+
+
+4. Ensure your profile is setup correctly from the command line by running the following [dbt commands](/reference/dbt-commands).
+
+
+    - [dbt compile](/reference/commands/compile) &mdash; generates executable SQL from your project source files
+    - [dbt run](https://docs.getdbt.com/reference/commands/run) &mdash; compiles and runs your project
+    - [dbt test](https://docs.getdbt.com/reference/commands/test) &mdash; compiles and tests your project
+    - [dbt build](https://docs.getdbt.com/reference/commands/build) &mdash; compiles, runs, and tests your project
+    - [dbt docs generate](/reference/commands/cmd-docs#dbt-docs-generate) &mdash; generates your project's documentation.
+    - [dbt docs serve](/reference/commands/cmd-docs#dbt-docs-serve) &mdash; starts a webserver on port 8080 to serve your documentation locally and opens the documentation site in your default browser.
+
+For complete details, refer to the [dbt command reference](/reference/dbt-commands).
+
+Here's what a successful output will look like:
+
+```jinja
+
+(venv) ➜  jaffle_shop_duckdb git:(duckdb) dbt build
+15:10:12  Running with dbt=1.8.1
+15:10:13  Registered adapter: duckdb=1.8.1
+15:10:13  Found 5 models, 3 seeds, 20 data tests, 416 macros
+15:10:13  
+15:10:14  Concurrency: 24 threads (target='dev')
+15:10:14  
+15:10:14  1 of 28 START seed file main.raw_customers ..................................... [RUN]
+15:10:14  2 of 28 START seed file main.raw_orders ........................................ [RUN]
+15:10:14  3 of 28 START seed file main.raw_payments ...................................... [RUN]
+....
+
+15:10:15  27 of 28 PASS relationships_orders_customer_id__customer_id__ref_customers_ .... [PASS in 0.32s]
+15:10:15  
+15:10:15  Finished running 3 seeds, 3 view models, 20 data tests, 2 table models in 0 hours 0 minutes and 1.52 seconds (1.52s).
+15:10:15  
+15:10:15  Completed successfully
+15:10:15  
+15:10:15  Done. PASS=28 WARN=0 ERROR=0 SKIP=0 TOTAL=28
+
+```
+To query data, some useful commands you can run from the command line: 
+
+- [`dbt show`](/reference/commands/show) &mdash; run a query against the data warehouse and preview the results in the terminal.
```

Comment:
Maybe suggest a command here, like `dbt show --select "raw_orders"`

Reply:
Also worth mentioning I had a very old dbt version on my computer that it used, and `dbt show` didn't exist at that time, so I had to upgrade dbt versions.